### ISSUE TITLE
Better documentation/examples

### ISSUE THREAD

**Issue by deuill** [deuill/go-php#4]
Current documentation is lacking, especially in terms of code examples. Bring the documentation up to standards.

**Comment by otraore** [comment]
This would be nice

**Comment by deuill** [comment]
Indeed, I'm gonna work on a slight clean-up of the code (especially for PHP5/PHP7-specific files) and get some examples in. In the meanwhile, the test files should provide some insight into using the library.
